source/Personality.cpp

```diff
@@ -78,7 +81,10 @@ namespace {
 		{"opportunistic", OPPORTUNISTIC},
 		{"target", TARGET},
 		{"marked", MARKED},
-		{"launching", LAUNCHING}
+		{"launching", LAUNCHING},
+		{"shy", SHY},
+		{"elusive", ELUSIVE},
+		{"roving", ROVING}
```

Comment:
```suggestion
		{"roving", ROVING},
```

Reply:
Shy: The ship attempts to avoid non-friendly vessels, retreating to a safe minimum distance if any come too close.
Elusive: Ship prioritizes maintaining cloak regardless of fuel cost. When attacked it will prioritize disengaging from combat and raising cloak.
Roving: A general purpose "mill about aimlessly" behavior to put a bit of life into fleets inhabiting empty systems. 

Shy and elusive, taken together, is intended ensure that when ion storms force ships to decloak they don't do so right under the player. 

Roving is meant to add movement to indigenous creatures who otherwise have no goal to pursue in the system where they're found.